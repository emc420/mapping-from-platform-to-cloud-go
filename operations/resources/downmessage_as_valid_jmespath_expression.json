{
    "type": "productConfigurationRequestFrame",
    "payload": {
      "transmissionPeriodOfTheStatusPeriod": "{{ command.input }}"
    }
}