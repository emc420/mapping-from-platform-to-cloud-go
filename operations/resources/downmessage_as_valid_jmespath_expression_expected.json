{
  "type": "productConfigurationRequestFrame",
  "payload": {
    "transmissionPeriodOfTheStatusPeriod": {
      "prop1": 10,
      "prop2": {
        "prop3": "value"
      }
    }
  }
}